docs/user/admin/settings.rst

```diff
@@ -824,3 +824,55 @@ To Re-enable Data Sources:::
       }
     }
 
+plugins.query.field_type_tolerance
+==================================
+
+Description
+-----------
+
+This setting controls whether preserve arrays. If this setting is set to false, then an array is reduced
+to the first non array value of any level of nesting.
+
+If you have an index with the following value for a field::
```

Comment:
doc is exectuable, you can run ppl/sql query directly. for example, https://github.com/opensearch-project/sql/blob/5dafb2fc5960e0a1edff70a089a5390519fc909c/docs/user/ppl/cmd/eval.rst

Reply:
I have reworked the documentation. It still does not actually run the queries, although they are run for other files such as `user/beyond/partiql.rst`.